pyproject.toml

```diff
@@ -85,11 +72,18 @@ select = [
   "RUF", # Ruff-specific rules
```

Comment:
The `select` section should now be under `[tool.ruff.lint]`

Reply:
thanks, I am going to change that in a bit. thanks for the update about that!